server/src/main/java/org/moparforia/server/Server.java

```diff
@@ -217,4 +223,36 @@ public void run() {
             }
         }
     }
+
+    /**
+     * Determines where to look for tracks.
+     * In order of priority:
+     * 1. If user has specified a tracks directory CLI flags, search for it in the default FileSystem
+     * 2. Else, if running in a jar file, use bundled tracks
+     * 3. Else, if running in an IDE, use resources folder
+     * 4. Else, use default filesystem and look in default directory
+     */
+    private TracksLocation getTracksLocation() throws URISyntaxException, IOException {
```

Comment:
Is it still possible to add custom tracks to the server and if so how?

Reply:
Yes, by using the CLI flag `--tracks-dir` when starting the server. I'll add some clarification around this to the README.